recommendation_engine/pmf/__init__.py

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
-"""This file defines a parameter class that contains the model hyperparameters.
```

Comment:
Un-rename this.

Reply:
Hello @rootAvish In this comment, you meant the comment line of the file, right?

Reply:
Deleted this file :)